### PR TITLE
[TACHYON-998] Get number of block locks from TachyonConf

### PR DESCRIPTION
**Pull request by miasantreble**
https://tachyon.atlassian.net/browse/TACHYON-998

### CODE REVIEW

**Comment by haoyuan** [comment]
Jenkins, test this please.

File: common/src/main/java/tachyon/Constants.java [hunk 1]
```diff
@@ -204,6 +204,7 @@
       "tachyon.worker.evict.strategy.lrfu.attenuation.factor";
   public static final String WORKER_MAX_TIERED_STORAGE_LEVEL =
       "tachyon.worker.tieredstore.level.max";
+  public static final String WORKER_LOCK_COUNT = "tachyon.worker.lock.count";
```
Comment (gpang) [inline common/src/main/java/tachyon/Constants.java]: This should probably have the word "block"?

**Comment by gpang** [comment]
I left a minor comment.

Also, documentation should be added to the docs to explain the new parameter. Thanks!

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by jsimsa** [comment]
LGTM +1

File: docs/Configuration-Settings.md [hunk 2]
```diff
@@ -406,6 +406,12 @@ number.
   <td>10000</td>
   <td>Timeout between worker and client connection indicating a lost session connection.  Specified in milliseconds</td>
 </tr>
+<tr>
+  <td>tachyon.worker.block.lock.count</td>
+  <td>1000</td>
+  <td>Total number of locks for a Tachyon block worker.</td>
```
Comment (gpang) [inline docs/Configuration-Settings.md]: I think this description should be more explicit that these locks are block locks. Maybe something like: "The number of block locks..." Also, what are the implications for larger number or fewer number of block locks? Maybe that tradeoff should also be in this description. Thanks!

**Comment by gpang** [comment]
Just a minor comment on the documentation. Otherwise, looks good!

**Comment by gpang** [comment]
LGTM

**Pull request merged**